src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -136,44 +155,22 @@ class PaymentMethodList extends Component {
 
         combinedPaymentMethods = _.map(combinedPaymentMethods, (paymentMethod) => ({
             ...paymentMethod,
-            onPress: (e) => this.props.onPress(e, paymentMethod.accountType, paymentMethod.accountData, paymentMethod.isDefault, paymentMethod.methodID),
-            iconFill: this.isPaymentMethodActive(paymentMethod) ? StyleUtils.getIconFillColor(CONST.BUTTON_STATES.PRESSED) : null,
-            wrapperStyle: this.isPaymentMethodActive(paymentMethod) ? [StyleUtils.getButtonBackgroundColorStyle(CONST.BUTTON_STATES.PRESSED)] : null,
+            onPress: (e) => onPress(e, paymentMethod.accountType, paymentMethod.accountData, paymentMethod.isDefault, paymentMethod.methodID),
+            iconFill: isPaymentMethodActive(actionPaymentMethodType, activePaymentMethodID, paymentMethod) ? StyleUtils.getIconFillColor(CONST.BUTTON_STATES.PRESSED) : null,
+            wrapperStyle: isPaymentMethodActive(actionPaymentMethodType, activePaymentMethodID, paymentMethod)
+                ? [StyleUtils.getButtonBackgroundColorStyle(CONST.BUTTON_STATES.PRESSED)]
+                : null,
             disabled: paymentMethod.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE,
         }));
 
         return combinedPaymentMethods;
-    }
+    }, [actionPaymentMethodType, activePaymentMethodID, bankAccountList, cardList, filterType, network, onPress, payPalMeData]);
 
     /**
-     * Dismisses the error on the payment method
-     * @param {Object} item
-     */
-    dismissError(item) {
-        const paymentList = item.accountType === CONST.PAYMENT_METHODS.BANK_ACCOUNT ? ONYXKEYS.BANK_ACCOUNT_LIST : ONYXKEYS.CARD_LIST;
-        const paymentID = item.accountType === CONST.PAYMENT_METHODS.BANK_ACCOUNT ? lodashGet(item, ['accountData', 'bankAccountID'], '') : lodashGet(item, ['accountData', 'fundID'], '');
-
-        if (!paymentID) {
-            Log.info('Unable to clear payment method error: ', item);
-            return;
-        }
-
-        if (item.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE) {
-            PaymentMethods.clearDeletePaymentMethodError(paymentList, paymentID);
-        } else {
-            PaymentMethods.clearAddPaymentMethodError(paymentList, paymentID);
-        }
-    }
-
-    /**
-     * @param {Object} paymentMethod
-     * @param {String|Number} paymentMethod.methodID
-     * @param {String} paymentMethod.accountType
-     * @return {Boolean}
+     * Show add first payment copy when payment methods are
```

Comment:
NAB since you aren't the original author, but can we make this sentence more coherent 😄

Reply:
Changed to: "Render placeholder when there are no payments methods"